src/dotnet/Chat.UI.Blazor/Components/ChatAudioPanel/RecorderToggle.razor

```diff
@@ -17,14 +17,25 @@
 
 @if (CanRecord) {
     <div class="@wrapperCls recorder-wrapper">
-        <ButtonRound Class="record-off-btn"
-                     Click="OnClick"
-                     @ontouchend="OnLongPressStop"
-                     @onmouseup="OnLongPressStop"
-                     @onlongpress="OnLongPressStart"
-                     data-long-press-delay="200">
-            <i class="icon-mic text-3xl font-normal"></i>
-        </ButtonRound>
+        @if (ScreenSize.IsNarrow()) {
+            <ButtonRound Class="record-off-btn"
+                         Click="OnClick"
+                         @ontouchend="OnLongPressStop"
+                         @onmouseup="OnLongPressStop"
+                         @onlongpress="OnLongPressStart"
+                         data-long-press-delay="200">
+                <img src="/logo-white.svg" alt="logo" class="recorder-icon">
```

Comment:
Looks like only button content depends on ScreenSize. I think better to put `@if` statement inside button.

Reply:
I tried, that doesn't work